.evergreen/pymongo/workload-executor.py

```diff
@@ -0,0 +1,95 @@
+import copy
```

Comment:
slight preference for `.evergreen/<language>` rather than custom naming (similar to [libmongocrypt](https://github.com/mongodb/libmongocrypt/tree/master/bindings)). We could even do `.evergreen/<language>/<named project>` for cases where you have multiple flavors (e.g. `motor`, `pymongo`)

Reply:
Done!